docs/guides/layers.md

```diff
@@ -139,8 +150,18 @@ napari.view_image(retina, name='retina', scale=[1,10,1,1])
 viewer.layers['retina'].scale = [1,10,1,1]
 ```
 
-![napari viewer with an image where all layers are scaled equally; when rotated, the image appears flat. By using console below the canvas and applying a scale factor to one of the dimensions, the image's volume becomes apparent.](images/scaling.webm)
-
+```{raw} html
+<figure>
+  <video width="100%" controls autoplay loop muted playsinline>
+    <source src="../../_static/images/scaling.webm" type="video/webm" />
+    <source src="../../_static/images/scaling.mp4" type="video/mp4" />
+    <img src="../../_static/images/scaling.png"
```

Comment:
```suggestion
    <source src="../_static/images/scaling.webm" type="video/webm" />
    <source src="../_static/images/scaling.mp4" type="video/mp4" />
    <img src="../_static/images/scaling.png"
```
Same as above.

Reply:
Wow that is very weird. How come they were playing for me even with the wrong path? Anyway, I fixed it. Please confirm if it works for you now 😄